src/components/ConnectButtons.tsx

```diff
@@ -1,16 +1,15 @@
 'use client';
 import { useAccount, useConnect, useDisconnect } from 'wagmi';
 
-const ConnectButtons = () => {
-  const { address, isConnected } = useAccount();
+const ConnectButtons = ({ text }: { text: string }) => {
```

Comment:
This should simply be `ConnectButton` as we don't show multiple buttons at any time.